packages/@aws-cdk/lambda-layer-kubectl/test/kubectl-layer.test.ts

```diff
@@ -7,7 +7,7 @@ test('synthesized to a layer version', () => {
   const stack = new Stack();
 
   // WHEN
-  new KubectlLayer(stack, 'MyLayer');
+  new KubectlLayer(stack, 'MyLayer', { kubectlVersion: '1.22' });
```

Comment:
```suggestion
  new KubectlLayer(stack, 'MyLayer', { kubectlVersion: '1.22.6' });
```

Reply:
Yep... tests will most likely have hardcoded value unless we create a dummy cluster that will expose this piece of information.